src/docfx/lib/LocalizationUtility.cs

```diff
@@ -72,11 +72,15 @@ public static string AddLeftToRightMarker(CultureInfo culture, string text)
             return repository is null ? null : TryRemoveLocale(repository.Remote, out _, out var remoteLocale) ? remoteLocale : null;
         }
 
-        public static bool TryGetContributionBranch(string branch, [NotNullWhen(true)] out string? contributionBranch)
+        public static bool TryGetContributionBranch(string? branch, [NotNullWhen(true)] out string? contributionBranch)
         {
-            if (branch.EndsWith("-sxs"))
+            if (branch != null && branch.EndsWith("-sxs"))
             {
                 contributionBranch = branch[0..^4];
+                if (contributionBranch == "master")
```

Comment:
this change is also suspicious. This method is used in a couple of other places and the change seems to be not backward compatible.